File: fizzbuzz.py
def fizzBuzz(number):
    i = 0
    answer = []
    while i < number:
        string = str(i)
        if i%3 == 0 and i%5 == 0:
            string = "FizzBuzz"
        elif i%3 == 0:
            string = "Fizz"
        elif i%5 == 0:
            string = "Buzz"
        answer.append(string)
        i += 1
    return answer

example = fizzBuzz(100)

for txt in example:
    print(txt)

File: ed/#03-03-07-2019/group-04/fizzbuzz.js
// Henrique
// Deangelo
//Julio

const fizzBuzz = (num) => {
  let numeros = []
  num.forEach(element => {
    let text = '';
    if (element % 3 == 0) {
      text = text + 'Fizz';
    }
    if (element % 5 == 0) {
      text = text + 'Buzz';
    }
    if (element % 3 != 0 && element % 5 != 0) {
      text = element;
    }
    numeros.push(text);
  });
  return numeros
}

numero = [];
for (let i = 1; i <= 100; i++) {
  numero.push(i);
}
numFizzBuzz = fizzBuzz(numero);

console.log(numFizzBuzz);